rabix-backend-local/src/main/java/org/rabix/backend/local/BackendCommandLine.java

```diff
@@ -323,4 +405,30 @@ private static Resources extractResources(Map<String, Object> inputs, ProtocolTy
     }
   }
 
+  private static Object createInputValue(String[] value, DataType inputType) {
```

Comment:
Logic for creating File and Directory values needs to be encapsulated into the concrete Bindings implementation.